Automattic/jetpack
# Shortcopy: update the fullscreen Jetpack setup dialog

Fixes #11537 
Fixes #11532

#### Changes proposed in this Pull Request:


* Update copy and remove elements that we want to leave behind.

## Fullscreen dialog
--------------

### Desktop

![desktop](https://user-images.githubusercontent.com/1041600/54455509-c7f66600-473a-11e9-8faf-de29b6b03d27.png)

### iPad

![ipad](https://user-images.githubusercontent.com/1041600/54455522-ccbb1a00-473a-11e9-8b91-e25ac1c9ad73.png)

### iPhone

![iphone](https://user-images.githubusercontent.com/1041600/54455526-cfb60a80-473a-11e9-98dc-70eb451da032.png)

## Small connect dialog (copy in screenshots is wrong but it's ok in code)
--------------

### Desktop

![desktop](https://user-images.githubusercontent.com/1041600/54465463-07807a80-475a-11e9-8089-88d03abbdf2a.png)

### iPad

![ipad](https://user-images.githubusercontent.com/1041600/54465474-10714c00-475a-11e9-8069-c961eef54bd8.png)

### iPhone

![iphone](https://user-images.githubusercontent.com/1041600/54465476-1404d300-475a-11e9-8275-857dc9af16b4.png)

#### Testing instructions:





* Install Jetpack in a new site, activate it in Plugins screen
* Fullscreen dialog should display in Plugins screen, with the new style
* Go to other screens. Small connect panel should display, with the new style

#### Proposed changelog entry for your changes:


* Not needed. Part of Shortcopy.
